src/sequences/OEIS.ts

```diff
@@ -1,10 +1,41 @@
 import type {ValidationStatus} from '../shared/ValidationStatus'
 import {modulo} from '../shared/math'
-import {SequenceExportModule, SequenceExportKind} from './SequenceInterface'
```

Comment:
Just by pattern matching, this sequence doesn't have SequenceExportKind removed here, as the other ones do.  This seems to stop the PR from working with OEIS.

Reply:
Yes, this is intentional. Since, according to the designs, OEIS is no longer a sequence in itself but a way to discover sequences in the sequence switcher, its functionality as a Paramable in the `seqMODULES` has been removed. A later PR, namely the one which implements the sequence switcher, reimplements it.